src/dfcx_scrapi/tools_ml/utterance_generator_utils.py

```diff
@@ -54,7 +58,15 @@ def __init__(
 
     @staticmethod
     def progress_bar(current, total, bar_length=50, type_="Progress"):
-        """Display progress bar for processing."""
+        """Display progress bar for processing.
+        
+        Args: 
+            current: (int) number for current iteration.
```

Comment:
@Greenford you can drop adding the Type in the doc strings and instead add Type Hinting directly in the method definition.

So this:
```
Args:
   current: (int) number for current iteration.
```

Becomes:
```
Args:
  current: number for current iteration.
```

And the function definition becomes:
```py
@staticmethod
def progress_bar(
  current: int,
  total: int,
  bar_length: int = 50, 
  type_: str = "Progress"):
<code here>
```